runtime/contract_update_validation_test.go

```diff
@@ -1750,21 +1750,6 @@ func TestRuntimeContractUpdateValidation(t *testing.T) {
 		assertContractRemovalError(t, err, "Test")
 	})
 
-	t.Run("Remove contract interface with enum", func(t *testing.T) {
```

Comment:
Maybe keep the test and adjust it accordingly

Reply:
There's nothing to adjust I think, this test is testing that removing a type requirement is not a valid update, which isn't a relevant thing to test anymore.

Reply:
Ah, I see, makes sense!